components/NavigationBar/index.js

```diff
@@ -5,33 +5,42 @@ import { FaHouse, FaPlus, FaChartPie } from "react-icons/fa6";
 function NavigationBar() {
   const router = useRouter();
 
-  const isActive = (path) => {
-    return router.pathname === path;
-  };
+ function isActive(path) {
+   if (!path) {
+     return;
+   }
+   return router.pathname === path;
+ }
```

Comment:
You always call the function with a parameter, so you don't really need to check if path is undefined.
```suggestion
 function isActive(path) {
   return router.pathname === path;
 }
```